### PR TITLE
User can comment

### PR DESCRIPTION
**Pull request by pedbr**
# PR for User can comment posts feature
```
As a visitor,
In order to communicate with other users,
I would like to be able to comment on their posts
```
PT: https://www.pivotaltracker.com/story/show/167288035

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
   <%= @post.user.name %>
-</p>
\ No newline at end of file
+</p>
+
+<h2>Comments</h2>
+<% @post.comments.each do |comment| %>
+  <p>
+    <strong>Commenter:</strong>
+    <%= current_user.name %>
```
Comment (oliverochman) [inline app/views/posts/show.html.erb]: will the commenter always be the current user?

File: features/user_can_comment.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Feature:
+"As a visitor,
```
Comment (oliverochman) [inline features/user_can_comment.feature]: fix the indentation for the user story and add a feature title on line 1

File: features/user_can_comment.feature [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    | Fat Bob  |
+    | Slim Jim |
+    And the following post exists
+    | title        | text          | user    |
+    | Codeacademy  | Smooth-ride   | Fat Bob |
+    And I am logged in as "Slim Jim"
+    And I visit the site
+    And I click "FLORUM"
+
+  Scenario: User can comment a post
```
Comment (oliverochman) [inline features/user_can_comment.feature:18]: I would add some sort of Sad Path scenario here

File: spec/models/comment_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (oliverochman) [inline spec/models/comment_spec.rb]: You need to add unit tests for the comment model

**Review by oliverochman — CHANGES_REQUESTED** [review]
Why haven't you added an association between comment and the user that creates it?

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (pedbr) [reply app/views/posts/show.html.erb]: Yes, when you create a comment you're logged in your account, so the author of the comment is the current user. Right?

File: features/user_can_comment.feature (hunk 2 above)
Comment (pedbr) [reply features/user_can_comment.feature]: Fixed

File: features/user_can_comment.feature (hunk 3 above)
Comment (pedbr) [reply features/user_can_comment.feature:18]: Sad path added

File: spec/models/comment_spec.rb (hunk 4 above)
Comment (pedbr) [reply spec/models/comment_spec.rb]: All fixed!

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (oliverochman) [reply app/views/posts/show.html.erb]: yes but you list all comments here, all users should be able to comment right

Comment (pedbr) [reply app/views/posts/show.html.erb]: of course, you're totally right!! (again)! It's fixed now!

File: spec/models/comment_spec.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  describe 'DB table' do
+    it { is_expected.to have_db_column :body }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :body }
+  end
+
+  describe 'Associations' do
+    it { is_expected.to belong_to :post }
```
Comment (oliverochman) [inline spec/models/comment_spec.rb:13]: have you tested the association between user and comment in the `User` model spec?

Comment (pedbr) [reply spec/models/comment_spec.rb:13]: all associations now tested!

**Review by oliverochman — APPROVED** [review]

**Pull request merged**